soh/src/overlays/actors/ovl_En_Ex_Item/z_en_ex_item.c

```diff
@@ -138,8 +138,11 @@ void EnExItem_WaitForObject(EnExItem* this, GlobalContext* globalCtx) {
             case EXITEM_BOMB_BAG_BOWLING:
                 this->unk_17C = func_8002EBCC;
                 if (gSaveContext.n64ddFlag) {
-                    this->giDrawId = GetItemModelFromId(
-                        GetRandomizedItemIdFromKnownCheck(RC_MARKET_BOMBCHU_BOWLING_FIRST_PRIZE, GI_BOMB_BAG_20));
```

Comment:
we can just use this, we don't need to do any particle stuff up here

Reply:
Done